app/assets/stylesheets/headers.scss

```diff
@@ -61,121 +61,149 @@ h1, h2, h3, h4, h5, h6 {
   }
 }
 
-/*
-
-// Extra small devices (portrait phones, less than 576px)
-@media (max-width: 575.98px) {
-  h1, .trix-content h1 {
-    @include font(f4);
-    padding-top: 5px;
-    padding-bottom: 5px;
-  }
+@mixin responsive-typography {
+  @include responsive-typography-scale-up;
+}
 
-  h2 {
-    @include font(f5);
-  }
+@mixin responsive-typography-scale-up {
```

Comment:
Sent you screenshot but this is scaling up too much at least for the carousel on the homepage. Thinking its only the hero text on homepage where we might scale the typography as rest of the site works for with h1-h6 as is.

Reply:
Ok, will remove the scaling here. Always interesting seeing how things look on other people's screens!